packages/puppeteer-core/src/util/Function.test.ts

```diff
@@ -35,49 +35,9 @@ describe('Function', function () {
   });
 
   describe('stringifyFunction', () => {
-    // prettier-ignore
-    it('should work', () => { /* eslint-disable @typescript-eslint/no-unused-expressions */
-      class K1 { m() {} }
-      class K2 { m () {} }
-      class K3 { async m() {} }
-      class K4 { async m () {} }
-
-      const variations = [
-        // Non-arrow functions.
-        function() {},
-        function () {},
-        function x() {},
-        function x () {},
-        async function() {},
-        async function () {},
-        async function x() {},
-        async function x () {},
-        // Concise methods.
-        new K1().m,
-        new K2().m,
-        new K3().m,
-        new K4().m,
-        // Arrow functions.
-        () => {},
-        // @ts-expect-error irrelevant
-        (a) => {a;},
-        // @ts-expect-error irrelevant
-        a => {a;},
-        async () => {},
-        // @ts-expect-error irrelevant
-        async (a) => {a;},
-        // @ts-expect-error irrelevant
-        async a => {a;},
-        // Generator functions
-        function*a() {},
-        function *a() {},
-        function* a() {},
-        // Async generator functions
-        async function*a() {},
-        async function *a() {},
-        async function* a() {},
-      ];
-
+    it('should work', async () => {
+      // eslint-disable-next-line @typescript-eslint/no-require-imports
+      const {variations} = require('../../../../function-fixture.cjs');
```

Comment:
Why can't we just use `await import`?

Reply:
We run the cjs version in tests so import wouldn’t work.

Reply:
Hm.. I tried it before switching to cjs but trying it again and it does work so maybe it was some other issue.